learner_pathway_progress/utilities.py

```diff
@@ -57,6 +62,45 @@ def get_pathway_snapshot(user, pathway_uuid):
     return result
 
 
+def get_learner_enterprises_for_course_run(user_id, course_run):
+    """
+    Get the enterprises associated with the learner and the course_run.
+    """
+    enterprise_uuids = EnterpriseCourseEnrollment.get_enterprise_uuids_with_user_and_course(user_id, course_run)
+    return enterprise_uuids
+
+
+def get_learner_enterprises_for_course(user, course):
+    """
+    Get the enterprises associated with the learner and the course.
+    """
+    course_runs = course.get('course_runs') or []
+    learner_enterprise_uuids = set()
+    for course_run in course_runs:
+        enterprise_uuids = EnterpriseCourseEnrollment.get_enterprise_uuids_with_user_and_course(
```

Comment:
can we use the function defined above `get_learner_enterprises_for_course_run` here?